docs/guides/entry_points.md

```diff
@@ -0,0 +1,90 @@
+:::
+
+
+### Second example: delivery pricing
+
+Let’s imagine another example: an e-commerce sells large, unusual or even fragile items. Items might not all be in stock, depending on factors like size, colors or other aspect. If you sell plants, you might want to confirm it’s still alive. 
+
+Pricing for custom deliveries depends on a multitude of factors (destination, need for temperature control, tax status) and your logistical partner conveniently offers an API to send quotes. The issue is: the response time of that API is slow. Therefore, you trigger a request as soon as you visitors confirm the contents of their basket and the country where they want it delivered, but before you have confirmed that the items are in stock, for example. 
+
+You also can easily imagine offering renting a car, or a valuable asset and needing a quote from your insurance company, based on many factors: information about the person renting, time and duration, etc. 
+
+Let‘s say you want to test asking for fast and cheap options. Or you could be testing a new provider. Alternatively, you might want to test asking both for a quote and let customers compate. Whatever is the change you want to test, you will need to decide which API call to trigger before the customer is guaranteed to see pricing because you have to wait for confirmation, say, that the items are in stock.
+
+In this case too, the assignment happens when confirming basket contents and the address, but the entry point should be set to a little later, when the customer sees a quote.
+
+::: note
+
+If one of the API integration fails, and some customers assigned to Treatment don’t see the pricing quote because of a bug, then you’ll observe an [asymetric traffic](/statistics/sample-ratio-mismatch.md).
+
+:::
+
+There are more cases where tests on your deliveries would need an Entry point. Let’s say you want to improve the box opening experience, include a note, etc. The assignment has to hapen when packaging; the entry point has to be set when the customer receives the note. There’s no reason to include customers who didn’t get their packages when measuring the impact of a hand-written note on their satisfaction.
+
+Not all logistical test require a separate Entry point. If you want to test a promotion that offers free delivery under certain condition, you can decide which customer sees that promotion when it becomes visible, not earlier.
+
+::: note
+
+Be mindful when testing promotions on anonymous users: you risk contaminating your experiment. If they get re-assigned by refreshing their browser, or opening a private window, some users will stumble on it, notice the discrepancy, and figure out the trick. They might leak that information. If prospective buyers see it, they will try. You are better off testing promotions on entities that buyers can’t change easily: established user account, phone number, postal address.
```

Comment:
I think this is outside the bounds of this guide and should be removed. We're hypothesizing abut a bunch of risks we don't have real knowledge about

Reply:
Removed